context/app/static/js/components/cells/AutocompleteEntity/AutocompleteEntity.tsx

```diff
@@ -38,7 +38,7 @@ function AutocompleteEntity({ targetEntity, setter }: AutocompleteEntityProps) {
   const { data, isLoading } = useAutocompleteQuery({ targetEntity, substring });
 
   // Include currently selected options to avoid invalid value errors in console
-  const options = selectedOptions.concat(data || []);
+  const options = selectedOptions.concat(data ?? []);
```

Comment:
Is this out of precaution against potentially falsy data or did you notice data that necessitated this?

Reply:
`data` is initially `undefined` when the page first loads. I added a `console.log(data)` call to demonstrate:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/eaaf1c03-7986-42eb-a4d9-e9c334e55fcb)